app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -17,164 +37,289 @@ const Lank = require('lib/surface/Lank')
+      const thangType = this.loader.getThangType(slug)
+      if (!thangType) {
+        return
+      }
+      const thangOptions = {
+        scaleFactorX: scaleX,
+        scaleFactorY: scaleY,
+        pos: { x, y },
+        stateChanged: true
+      }
+
+      // If this background is already in use, just update thang instead of
+      // completely replacing the lank.
+      if (this.lanks[BACKGROUND] && this.lanks[BACKGROUND].thangType) {
+        if (this.lanks[BACKGROUND].thangType.get('slug') === slug) {
+          const thang = this.lanks[BACKGROUND].thang
+          _.merge(thang, thangOptions)
+          return
+        }
+      }
+
+      const backgroundThang = createThang(thangOptions)
+      this.addLank(BACKGROUND, thangType, backgroundThang)
+    })
   }
 
   /**
-   * Updates the left and right lank if they exist.
+   * Updates lanks
    * @param {bool} frameChanged - Needs to be true for Lank updates to occur.
    */
   update (frameChanged) {
-    this.left && this.left.update(frameChanged)
-    this.right && this.right.update(frameChanged)
+    Object.values(this.lanks)
+      .forEach(lank => lank.update(frameChanged))
+  }
+
+  /**
+   * Removes and cleans up resources for the provided lank.
+   * @param {string} key unique key for given lank.
+   */
+  removeLank (key) {
+    const lank = this.lanks[key]
+    if (!lank) { return }
+    lank.layer.removeLank(lank)
+    delete this.lanks[key]
+    lank.destroy()
   }
 
   /**
-   * Adds a lank to the given side.
-   * Needs to be able to check if there is an existing lank and handle appropriately.
-   * If a lank is created, it's always created offscreen.
+   * Adds a lank to the screen.
    *
-   * TODO: Handle existing lank
+   * If the lank already exists this method is a noop.
+   * Otherwise if there is a conflicting lank it is removed and replaced.
+   * Automatically flips a lank if they key is `right` in order to accomodate for
+   * right hand side characters.
    *
-   * @param {'left'|'right'} side
+   * @param {string} key
    * @param {Object} thangType
-   * @param {Object} systems
+   * @param {Object|undefined} thang?
    */
-  addLank (side, thangType) {
-    assertSide(side)
-    const thang = side === 'left'
-      ? createThang({ pos: {
-        x: this.stageBounds.topLeft.x - 2,
-        y: this.stageBounds.bottomRight.y
+  addLank (key, thangType, thang = undefined) {
+    // Handle a duplicate thangType with the same key.
+    if (this.lanks[key] && this.lanks[key].thangType) {
+      const original = this.lanks[key].thangType.get('original')
+      if (thangType.get('original') === original) {
+        // It's the same thangType. Don't add a new Lank.
+        return
+      } else {
+        // It's a lank we want to replace.
+        this.removeLank(key)
       }
+    }
+
+    // Initial coordinates for thangs being created offscreen.
+    // This feels like it could be refactored to be nicer.
+    if (key === 'right' && !thang) {
```

Comment:
Do we need these  default placements? Maybe we want to add support for the cinematic designers to set a creation point. (`moveLankInstantlyCommand`) and then set a desired animation target  position  with `moveLank`?